draft-nygren-httpbis-httpssvc.md

```diff
@@ -481,6 +481,14 @@ Because HTTPSSVC is received over an often insecure channel (DNS),
 clients MUST NOT place any more trust in this signal than if they
 had received a 302 redirect over cleartext HTTP.
 
+If the HTTPSSVC query results in a SERVFAIL error, the client SHOULD abandon
+the connection attempt and display an error message.  A SERVFAIL error can
+occur if the domain is DNSSEC-signed, the recursive resolver is
+DNSSEC-validating, and an active attacker between the recursive resolver
+and the authoritative resolver is attempting to prevent the upgrade to
+HTTPS.  Similarly, if the client validates DNSSEC, it SHOULD abandon the
+connection attempt if the HTTPSSVC RR fails to validate.
+
```

Comment:
Presumably this could also happen for servers that are misbehaving and don't understand the HTTPSSVC record type in the query?  I think we'd need more data an implementation experience.  Perhaps we could reword this as a MAY?  (Or put in Security Considerations?)

Reply:
Yeah, it depends on how much faith the client has in their resolver.  I would be OK with conditioning this on a secure transport to the resolver (as a proxy for "decent resolver, won't SERVFAIL on new RRTYPEs").

Reply:
OK, I've updated this PR to only apply to "cryptographically protected connections".  Please review.

Reply:
Looks fine now.